app/queries/undrawn_suites_query.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+#
+# Query to return the suites that don't belong to any draw. This can bee passed
+# an existing relation for a subset of all Suites.
+class UndrawnSuitesQuery
```

Comment:
I don't know if I like the drawn/undrawn naming -- I think drawless might be more appropriate here?